### ISSUE TITLE
Internet explorer opacity of landscape images 

### ISSUE THREAD

**Issue by jamieisboss** [ed-lea/jquery-collagePlus#10]
Landscape images do not have opacity set to 1 after plugin is loaded in internet explorer. in 0.3.2

Works fine in 0.3.1 and 0.3.2 works in firefox or chrome.

Bug in ie:

![iebug](https://f.cloud.github.com/assets/5108847/1009567/e7190426-0b3a-11e3-9143-8cec45910951.png)

fine in chrome:

![chrome_nubug](https://f.cloud.github.com/assets/5108847/1009575/f428d182-0b3a-11e3-8a4e-970433e9fc6d.png)

**Comment by ed-lea** [comment]
ok, thanks for the report. looking into it. Are you able to use 0.3.1 for the time being? Also, do you have the HTML for the collage area, trying to see if the <img> is wrapped in any other element.

**Comment by jamieisboss** [comment]
I'm using 0.3.1 for now and it's working great thanks!

the html looks something like this:

```
<div class="image-grid" >
for each image:
<a href="image_permalink"><img src="http://image_url" alt="alt" /></a>
</div>
```

It's in wordpress and I've stripped the php out but basically it renders an image wrapped in an anchor for every image on the page. All of that is in a div called image-grid.

Thanks for the plugin by the way, it's awesome!

**Comment by david-wilson** [comment]
Using Chrome 28.0.1500.95 on Ubuntu, I'm seeing the same issue jamieisboss is seeing.  Rolling backing to 0.3.1 fixes the issue for me.

**Comment by ed-lea** [comment]
Hi Folks, unfortunately my machine is on a boat to the US so I don't have access to windows or ubuntu. I have made a branch to try and fix this and I wondered if you have a chance if you could try this commit https://github.com/ed-lea/jquery-collagePlus/blob/69c66fcc6620ac4a4aa5f0f96b0ad6b253570294/jquery.collagePlus.js and see if it makes any difference.

**Comment by david-wilson** [comment]
Unfortunately, that commit does not appear to make a difference in my case.  My HTML (Django template):

``` html
<div class="Collage">
    {% for photo in photo_list %}
         {% if photo.thumbnail and photo.web_size %}
         <a href="{{ photo.web_size.url }}" class="photo"><img src="{{ photo.thumbnail.url }}" width="{{ photo.thumb_width }}" height="{{ photo.thumb_height }}" alt=""/></a>
         {% endif %}
    {% empty %}
        <div>No photos yet.<div>
    {% endfor %}
</div>
```

I am using the remove whitespace plugin as well.

**Comment by ed-lea** [comment]
please reopen if this issue is still occurring. I can't seem to replicate it.

**Comment by matteoraf** [comment]
Same issue happening to me. Solved using 0.3.1.
